### PR TITLE
Enhance 3.x testing

### PR DESCRIPTION
**Pull request by yann-eugone**
- Standardise sources
- Update testing matrix to support PHP 8.0 & Symfony 5.3

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -16,12 +16,12 @@ jobs:
         include:
           - php-version: 7.1
             symfony-version: 4.4.*
-          - php-version: 7.4
+          - php-version: 8.0
```
Comment (J-Ben87) [inline .github/workflows/tests.yml:19]: Why did you remove the support for php 7.4?

File: src/Event/SitemapAddUrlEvent.php [hunk 2]
```diff
@@ -15,7 +15,11 @@
 use Symfony\Contracts\EventDispatcher\Event;
 
 /**
- * Event to allow generation of static routes sitemap urls.
+ * Event called whenever an static url is about to be added to sitemap.
```
Comment (J-Ben87) [inline src/Event/SitemapAddUrlEvent.php]:
```suggestion
 * Event called whenever a static url is about to be added to sitemap.
```

File: src/Event/SitemapPopulateEvent.php [hunk 3]
```diff
@@ -15,9 +15,10 @@
 use Symfony\Contracts\EventDispatcher\Event;
 
 /**
- * Manage populate event
+ * Event called whenever a sitemap build is asked.
```
Comment (J-Ben87) [inline src/Event/SitemapPopulateEvent.php]:
```suggestion
 * Event called whenever a sitemap build is requested.
```

File: src/EventListener/StaticRoutesAlternateEventListener.php [hunk 4]
```diff
@@ -8,6 +17,11 @@
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
 
+/**
+ * Listen to "presta_sitemap.add_url" event.
+ * Decorate translatable Url with multi-lang alternates.
```
Comment (J-Ben87) [inline src/EventListener/StaticRoutesAlternateEventListener.php]:
```suggestion
 * Decorate translatable Url with multi-lang alternatives.
```

File: src/Sitemap/Url/GoogleMultilangUrlDecorator.php [hunk 5]
```diff
@@ -14,11 +14,9 @@
 use Presta\SitemapBundle\Sitemap\Utils;
 
 /**
- * Decorate w/ google alternate language url guidelines
- *
- * @see http://support.google.com/webmasters/bin/answer.py?hl=en&answer=2620865
+ * Url decorator to add alternates to a sitemap Url.
```
Comment (J-Ben87) [inline src/Sitemap/Url/GoogleMultilangUrlDecorator.php]:
```suggestion
 * Url decorator to add alternatives to a sitemap Url.
```

File: tests/Integration/config/packages/5.3/framework.yaml [hunk 6]
```diff
@@ -0,0 +1,17 @@
+framework:
+    secret: '%env(APP_SECRET)%'
+    #csrf_protection: true
```
Comment (J-Ben87) [inline tests/Integration/config/packages/5.3/framework.yaml]: Is it necessary to keep this commented line?

File: tests/Integration/config/packages/5.3/framework.yaml [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+    # Enables session support. Note that the session will ONLY be started if you read or write from it.
+    # Remove or comment this section to explicitly disable session support.
+    session:
+        handler_id: null
+        cookie_secure: auto
+        cookie_samesite: lax
+        storage_factory_id: session.storage.factory.mock_file
+
+    #esi: true
```
Comment (J-Ben87) [inline tests/Integration/config/packages/5.3/framework.yaml]: Is it necessary to keep these commented lines?

File: .github/workflows/tests.yml (hunk 1 above)
Comment (yann-eugone) [reply .github/workflows/tests.yml:19]: The bundle is still compatible with PHP 7.4, this is all about having a shorter testing matrix : only lowest and highest versions tested

Do you think it can be error prone ?

File: tests/Integration/config/packages/5.3/framework.yaml (hunk 6 above)
Comment (yann-eugone) [reply tests/Integration/config/packages/5.3/framework.yaml]: Not at all, came from the recipe I copied from Symfony's repository.

I will clean these

File: .github/workflows/tests.yml (hunk 1 above)
Comment (J-Ben87) [reply .github/workflows/tests.yml:19]: Not at all, I was just wondering why you were removing that version which I was considering as a major version.

However I like keeping only the lowest and highest versions. Or maybe you could keep the lowest and highest for each major? Like `7.1` and `7.4`, and `8.0`... until the next minor is released?

**Review by J-Ben87 — APPROVED** [review]

File: .github/workflows/tests.yml (hunk 1 above)
Comment (yann-eugone) [reply .github/workflows/tests.yml:19]: Actually, there is no need to keep `7.4` in testing matrix, the idea was to keep lowest and highest PHP version for each lowest and highest Symfony version
